packages/devtools/web/main.dart

```diff
@@ -14,9 +14,13 @@ void main() {
   final PerfToolFramework framework = PerfToolFramework();
 
   if (!browser.isChrome) {
+    final browserName =
+        // Edge shows up as IE, so we replace it's name too to avoid confusion.
```

Comment:
nit: remove 'too'

Reply:
Whoops, must be a bug in my keyboard 😄

Thanks!